storage/diskmetricstore_test.go

```diff
@@ -927,9 +927,9 @@ func TestAddDeletePersistRestore(t *testing.T) {
 		newPushFailedTimestampGauge(grouping5, time.Time{}).Metric[0],
 		newPushFailedTimestampGauge(grouping6, time.Time{}).Metric[0],
 	)
-	mfLaballed := proto.Clone(mfUnlabelled).(*dto.MetricFamily)
-	// sanitizeLabels should add these labels to the unlabelled metric
-	mfLaballed.Metric[0].Label = []*dto.LabelPair{
+	mfLabelled := proto.Clone(mfUnlabelled).(*dto.MetricFamily)
+	// SanitizeLabels should add these labels to the unlabelled metric.
```

Comment:
Actually, sanitizeLabels is an actual method name. So I complained too soon. But anyway, we don't have to get even nitpickier.